### PR TITLE
feat: skillset distribution plot

### PR DESCRIPTION
**Pull request by n0w0f**
## Summary by Sourcery

Add a skillset distribution plot to the appendix to visualize and describe the distribution of required skills in the ChemBench corpus, enhancing the documentation with a new figure and accompanying text.

New Features:
- Introduce a skillset distribution plot to visualize the distribution of required skills in the ChemBench corpus.

Documentation:
- Add a figure and description to the appendix to illustrate the composition of required skills in the ChemBench corpus.

### CODE REVIEW

**Comment by kjappelbaum** [comment]
looks fancy! are you our new figure wizard, @n0w0f?

File: src/tex/appendix.tex [hunk 1]
```diff
@@ -125,7 +125,16 @@ \subsubsection{Composition}
 
 \normalsize
 
-In addition, as shown in \Cref{tab:chembench_corpus_cognitive} the \chembench corpus can be divided considering the different skills needed to solve the questions.
+
+In addition, as shown in \Cref{tab:cb_skillset} the \chembench corpus can be divided considering the different skills needed to solve the questions. The plot shows an even distribution of the required skills in the \chembench corpus. \Cref{tab:cb_skillset} shows an example question for each skill category.
+
```
Comment (kjappelbaum) [inline src/tex/appendix.tex:130]:
```suggestion
In addition, as shown in \Cref{tab:cb_skillset} the \chembench corpus can be divided considering the different skills needed to solve the questions. The plot shows a balanced distribution of the required skills in the \chembench corpus. \Cref{tab:cb_skillset} shows an example question for each skill category.

```

File: src/tex/appendix.tex [hunk 2]
```diff
@@ -125,7 +125,16 @@ \subsubsection{Composition}
...
+\begin{figure}
+    \centering
+    \includegraphics[width=\textwidth]{figures/CB_SKILL_FIG_V1.pdf}
+    \caption{\textbf{Composition of the required skills considered in the \ChemBench corpus.} The circular plot shows the distribution of required skills in the \chembench corpus.}
```
Comment (kjappelbaum) [inline src/tex/appendix.tex:134]:
```suggestion
    \caption{\textbf{Composition of the required skills considered in the \chembench corpus.} The circular plot shows the distribution of required skills in the \chembench corpus.}
```

not sure if latex commands are case insensitive

**Comment by n0w0f** [comment]
> looks fancy! are you our new figure wizard, @n0w0f?

if only i could script to do this.

**Comment by kjappelbaum** [comment]
> > looks fancy! are you our new figure wizard, @n0w0f?
> 
> if only i could script to do this.

apple script or the new computer mode from anthropic ;)

**Pull request merged**